app/models/feedback.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class Feedback < ApplicationRecord
+  belongs_to :user
+
+  validates :description, presence: true, length: { maximum: 500 }
```

Comment:
```suggestion
  validates :description, presence: true, length: { maximum: 5000 }
```

500 is too small, if they want it to be big let it be like that